packages/webview_flutter/webview_flutter_wkwebview/lib/src/webkit_webview_controller.dart

```diff
@@ -530,6 +530,16 @@ class WebKitWebViewController extends PlatformWebViewController {
   ) async {
     _onPermissionRequestCallback = onPermissionRequest;
   }
+
+  /// Whether to enable the platform's webview content debugging tools.
```

Comment:
As stated in the other comment, I believe this flag only enables debugging for the `WebView` it is called on. This should include this.